Scripts/RemediationScripts/Readme.md

```diff
@@ -1,4 +1,54 @@
-## Load remediation script to fix failed controls of Azure Tenant Security Solution - Step by Step
+# Bulk Remediation Script (BRS)
+
+## On this page:
+
+- [Overview](README.md#overview)
+- [BRS supported control list](README.md#brs_supported_control_list)
+- [How to use BRS](README.md#how_to_use_brs)
+
+-----------------
+
+## **Overview**
+Bulk remediation scripts can be used to remediate non-compliant resources/subscription for a control. These scripts are limited to the controls that have relatively lower risk to auto-remediate.
+
+## **BRS supported control list**
+|Control Id|Control Name|Link to BRS|Minimum permissions required to run the script|Supports managed identity based remediation**|Supports rollback?|
+|---|---|---|---|---|---|
+|Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN|Enterprise applications using APIM must authenticate developers/applications using Azure Active Directory backed credentials|[Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices](Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1)|At least contributor at resource level|Restricted to 'User' account type.|No|
```

Comment:
Skip the phrase "At least" since the heading says that "minimum permissions" (Use something like Contributor role at resource level). Similarly for the others.

Reply:
Done.